tests/test.c

```diff
@@ -4631,9 +4631,29 @@ void test_flush(rd_kafka_t *rk, int timeout_ms) {
                           rd_kafka_outq_len(rk));
 }
 
+int test_is_deprecated_conf_group_protocol_consumer(const char *name) {
```

Comment:
Let's change the naming to `forbidden` because they're not deprecated still but they cannot be used with consumer protocol.
```suggestion
int test_is_forbidden_conf_group_protocol_consumer(const char *name) {
```